src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -45,12 +46,22 @@ public override BigInteger TotalSupply(StoreView snapshot)
 
         protected override void OnBalanceChanging(ApplicationEngine engine, UInt160 account, NeoAccountState state, BigInteger amount)
         {
-            DistributeGas(engine, account, state);
+            var m = ProtocolSettings.Default.CommitteeMembersCount - ProtocolSettings.Default.ValidatorsCount;
+            if (state.BalanceHeight < engine.Snapshot.PersistingBlock.Index - 2 * m)
+                DistributeGas(engine, account, state);
```

Comment:
What does this mean?

Reply:
Avoid unprofitable calculation, especially within a epoch.

Reply:
NEO holders get incentive every block, right?

Reply:
Yes. 
I mean the voting bonus part.

Reply:
But if I'm a NEO holder, and if my balance of NEO changes in an epoch, I won't get any incentive.

Reply:
Removed.